Exercise10Keenan.R

```diff
@@ -0,0 +1,88 @@
+#Problem 1: Cumulative score graphing
+library(ggplot2)
+#Load in data
+rawData <- read.table('UWvMSU_1-22-13.txt', header = TRUE)
+
+#create cumulative score variables and dataframes
+UWCumScore <- 0
+MSUCumScore <- 0
+uw <- data.frame(matrix(ncol = 2, nrow = 0))
+msu <- data.frame(matrix(ncol = 2, nrow = 0))
+first_entry <- data.frame(0,0)
+names(first_entry)<-c('time', 'score')
+cn <- c("time", "score")
+colnames(uw) <- cn
+colnames(msu) <- cn
+#put in start score of 0-0
+uw<-rbind(uw, first_entry)
+msu<-rbind(msu, first_entry)
+

```

Comment:
or you can do something like the code below. This code is equivalent to from line 9 to 18:
uw <- data.frame(time = 0, score = 0)
msu <- data.frame(time = 0, score = 0)